doc/nrf/releases_and_maturity/migration/migration_hwmv2.rst

```diff
@@ -0,0 +1,36 @@
+.. _hwmv1_to_v2_migration:
+
+Migrating to the current hardware model
+#######################################
+
+The *hardware model* is the manner in which SoCs and boards are named, defined, and used.
+Starting with |NCS| version 2.7.0, the hardware model has been overhauled.
+
+This section describes how to migrate from the hardware model used in |NCS| versions 2.6.0 and earlier (commonly referred to as "hardware model v1")  to the current one (commonly referred to as "hardware model v2").
+
+.. note::
+   The current hardware model was introduced in the Zephyr Project shortly after the release of 3.6.0.
+
+The current hardware model adds support for features like the following:
+
+* Support for multi-core, multi-architecture AMP (Asymmetrical Multi Processing) SoCs.
+* Support for multi-SoC boards.
+* Support for reusing the SoC and board Kconfig trees outside of the Zephyr and |NCS| build system.
+* Support for advanced use cases with :ref:`zephyr:sysbuild`.
+* Removal of all existing arbitrary and inconsistent uses of Kconfig and folder names.
+
+For more information about the rationale, development, and concepts behind the new model, see the `Zephyr original issue <Zephyr issue #51831>`_, the `Zephyr original Pull Request <Zephyr pull request #50305>`_,.
+For the complete set of changes introduced, consult the `hardware model v2 commit <Zephyr commit 8dc3f8>`_.
+
+Some non-critical features, enhancements, and improvements of the new hardware model are still in development. Check the `Zephyr hardware model v2 enhancements issue <Zephyr issue #69546>`_ for a complete list.
+
+For |NCS| users that also have out-of-tree boards, the transition from hardware model v1 to hardware model v2 is optional, because |NCS| provides full backwards compatibility for any existing out-of-tree boards.
+
+If you choose not to convert the out-of-tree board, you will receive a warning when building, informing you that the board definition is in the format compatible with hardware model v1.
+
+If you choose to convert your board from hardware model v1 to v2, consult the following pages:
+
+* :ref:`zephyr:hw_support_hierarchy` and the subsequent sections to understand the model in detail.
+* Use the `ncs-example-application conversion commit <ncs-example-application commit f9f2da>`_ as an example on how to port a simple board.
+* Additionally, a `conversion script <hwmv2 conversion script>`_ is available and works reliably in many cases (though multi-core SoCs may not be handled entirely).
```

Comment:
A note from the UX testing: 

Referring to the PR for HWMv2 migration guide, it refers to a commit for the custom_plank sample converting it to HWMv2 as a reference, which is good. But I miss an example using the conversion script (board_v1_to_v2.py). A simple line showing all the parameters needed to run the script on the custom_plank + an overview of which additional changes are needed after the script has run would be very helpful.   


Could you add something like this?